app/controllers/api/base_controller.rb

```diff
@@ -102,6 +102,10 @@ def options
       render_options(@req.collection)
     end
 
+    def single_resource?
+      !@req.json_body.key?('resources')
+    end
+
```

Comment:
I think this method should probably go in lib/api/request_adapter.rb,  WDYT @jntullo ?

Reply:
@abellotti yeah, i agree with that

Reply:
@abellotti @jntullo Done :+1: